plugins/datadog_tracer/config/initializers/datadog_tracer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+if SamsonDatadogTracer.enabled?
+
+  SamsonDatadogTracer::IGNORED_URLS = Set[
+    '/ping',
+    '/streams',
```

Comment:
streams is dead ... maybe `/cable` ?
... also this setting should be in the base ?

Reply:
nvm about the base comment ...